packages/calcite-components/src/components/action-menu/action-menu.e2e.ts

```diff
@@ -429,8 +429,8 @@ describe("calcite-action-menu", () => {
       expect(actions[1].getAttribute(activeAttr)).toBe(null);
       expect(actions[2].getAttribute(activeAttr)).toBe(null);
 
-      await page.keyboard.press("Tab");
-
+      const button = await page.find("button");
+      button.focus();
```

Comment:
> Also, wouldn't pressing tab still cause the menu to blur?

Yes.

Reply:
Could you expand on that? I'm asking since we use the tab key to trigger blurring in other tests.

Reply:
Yeah, It just makes more sense to rely on the blur event rather than a key event because the blur event can occur if another element/component steals focus whereas the tab is relying only on that keyboard event.

Reply:
I should clarify, I'm talking specifically about the test changes, not the component change to use `blur` vs `tab` key.

Reply:
I just changed from tab to blur since there is some kind of timing issue with the tab I think.